packages/core/src/configuration.ts

```diff
@@ -74,6 +76,12 @@ export function buildConfiguration(userConfiguration: UserConfiguration, buildEn
   }
 
   const configuration: Configuration = {
+    isEnabled: (feature: string) => {
+      return (
+        Array.isArray(configuration.enableExperimentalFeatures) &&
+        includes(configuration.enableExperimentalFeatures, feature)
```

Comment:
It could be cleaner to not expose `configuration.enableExperimentalFeatures` and use `userConfiguration` directly here
```suggestion
        Array.isArray(userConfiguration.enableExperimentalFeatures) &&
        includes(userConfiguration.enableExperimentalFeatures, feature)
```